ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/server/KsqlRestApplicationTest.java

```diff
@@ -354,51 +347,6 @@ public void shouldSendCreateStreamRequestBeforeSettingReady() {
     inOrder.verify(serverState).setReady();
   }
 
-  @Test
```

Comment:
As we no longer check preconditions from the rest application, we no longer need these tests.